articles/mysql/flexible-server/how-to-data-in-replication.md

```diff
@@ -80,110 +80,112 @@ The following steps prepare and configure the MySQL server hosted on-premises, i
 
 1. Set the source server to read-only mode.
 
-Before starting to dump out the database, the server needs to be placed in read-only mode. While in read-only mode, the source will be unable to process any write transactions. Evaluate the impact to your business and schedule the read-only window in an off-peak time if necessary.
+   Before starting to dump out the database, the server needs to be placed in read-only mode. While in read-only mode, the source will be unable to process any write transactions. Evaluate the impact to your business and schedule the read-only window in an off-peak time if necessary.
 
-```sql
-FLUSH TABLES WITH READ LOCK;
-SET GLOBAL read_only = ON;
-```
+   ```sql
+   FLUSH TABLES WITH READ LOCK;
+   SET GLOBAL read_only = ON;
+   ```
 
 1. Get binary log file name and offset.
 
    Run the [`show master status`](https://dev.mysql.com/doc/refman/5.7/en/show-master-status.html) command to determine the current binary log file name and offset.
 
    ```sql
-      show master status;
+   show master status;
    ```
 
-The results should appear similar to the following. Make sure to note the binary file name for use in later steps.
+   The results should appear similar to the following. Make sure to note the binary file name for use in later steps.
 
-:::image type="content" source="./media/how-to-data-in-replication/master-status.png" alt-text="Master Status Results":::
+   :::image type="content" source="./media/how-to-data-in-replication/master-status.png" alt-text="Master Status Results":::
 
 ---
 
 ## Dump and restore the source server
 
 1. Determine which databases and tables you want to replicate into Azure Database for MySQL flexible server and perform the dump from the source server.
 
-    You can use mysqldump to dump databases from your primary server. For details, refer to [Dump & Restore](../concepts-migrate-dump-restore.md). It's unnecessary to dump the MySQL library and test library.
+   You can use mysqldump to dump databases from your primary server. For details, refer to [Dump & Restore](../concepts-migrate-dump-restore.md). It's unnecessary to dump the MySQL library and test library.
 
-2. Set source server to read/write mode.
+1. Set source server to read/write mode.
 
    After the database has been dumped, change the source MySQL server back to read/write mode.
 
    ```sql
    SET GLOBAL read_only = OFF;
    UNLOCK TABLES;
    ```
-[!NOTE]
-> Before the server is set back to read/write mode, you can retrieve the GTID information using global variable GTID_EXECUTED. The same will be used at the later stage to set GTID on the replica server
 
-3. Restore dump file to new server.
+   >[!NOTE]
+   > Before the server is set back to read/write mode, you can retrieve the GTID information using global variable GTID_EXECUTED. The same will be used at the later stage to set GTID on the replica server
```

Comment:
```suggestion
   > Before the server is set back to read/write mode, you can retrieve the GTID information using global variable GTID_EXECUTED. This will be used at the later stage to set GTID on the replica server.
```